server/app/mutations/grid_certificates/common.rb

```diff
@@ -46,5 +46,13 @@ def get_authz_for_domain(grid, domain)
     def acme_endpoint
       ENV['ACME_ENDPOINT'] || ACME_ENDPOINT
     end
+
+    def resolve_service(grid, service_name)
+      stack_name, service = service_name.split('/')
```

Comment:
What about `--linked-service stackless-lb`?

Reply:
IMO the API should always take in (and out) real "paths" and possible nullification/de-nullification should happen on the CLI side. So if we want to support `--linked-service stackless-lb`, CLI should resolve that to `null/stackless-lb`

Reply:
SGTM

Reply:
done on CLI side